sympy/concrete/summations.py

```diff
@@ -492,6 +492,16 @@ def is_convergent(self):
             except NotImplementedError:
                 pass
 
+        ### ----------- raabe test ---------------- ###
+        lim_val = Limit(sym * (sequence_term/sequence_term.subs(sym, sym + 1))
```

Comment:
`abs` should probably be added here since Raabe's test is for series with positive terms.